README.md

```diff
@@ -1,2 +1,88 @@
-# jesi_dart_analysis
-This package provides lint rules for Dart and Flutter which are used at JESI
+# JESI Dart Analysis
```

Comment:
https://github.com/jesims/jesi_dart_analysis/blob/JESI-5570/README.md

Reply:
Assuming you added a link to view the non-raw file.

Reply:
Yup